sql/moz-fx-data-shared-prod/telemetry_derived/clients_last_seen_event_v1/query.sql

```diff
@@ -0,0 +1,51 @@
+WITH _current AS (
+  SELECT
+    CAST(n_logged_event > 0 AS INT64) AS days_logged_event_bits,
+    CAST(n_created_pictureinpicture > 0 AS INT64) AS days_used_pictureinpicture_bits,
+    CAST(n_viewed_protection_report > 0 AS INT64) AS days_viewed_protection_report_bits,
+    * EXCEPT (submission_date)
+  FROM
+    clients_daily_event_v1
+  WHERE
+    submission_date = @submission_date
+),
+_previous AS (
+  SELECT
+    days_logged_event_bits,
+    days_used_pictureinpicture_bits,
+    days_viewed_protection_report_bits,
+    * EXCEPT (
+      days_logged_event_bits,
+      days_used_pictureinpicture_bits,
+      days_viewed_protection_report_bits,
+      submission_date
+    )
```

Comment:
Wouldn't this be the same as?

```sql
_previous AS (
  SELECT
    * EXCEPT (
      submission_date
    )
  FROM
...
```

Reply:
as fields are added this will change the order of the fields so that they match between `_current` and `_previous`, which is necessary for the `IF(..., _current, _previous).*` clause.

Reply:
I'll document this in a comment. It's indeed surprising to see.